### ISSUE TITLE
FEATURE REQUEST: MAKE THIS THEME ONE LINER LIKE SAFARI 15

### ISSUE THREAD

**Issue by kb12000** [easonwong-de/White-Sur-Firefox-Theme-MacOS#7]
PLEASE MAKE A MONTEREY VERSION FOR WINDOWS IF POSSIBLE

**Comment by easonwong-de** [comment]
I also want that, badly :(

**Comment by easonwong-de** [comment]
For me, the most difficult part is how to make active bar turn into URL field when clicked. I’ve seen a demo before, but it was way too complicated, so it might stop working with any Firefox update.

**Comment by kb12000** [comment]
Can you make the theme without active bar turning into url…just make tabs and url bar in one line like this 
https://raw.githubusercontent.com/vinceliuice/WhiteSur-gtk-theme/pictures/pictures/firefox.svg

**Comment by CREATORCorp** [comment]
Yep! Looking for the same thing. I can't find a good theme like that. Monterey style one liner looks the best and minimalist with bookmark hover. I hope u can do that, because i'm too dumb to make this work. Thx for the work, the theme is good, need some minor fixes.
